navigation/stack_navigators/StoresStack.js

```diff
@@ -0,0 +1,49 @@
+import {
+  CardStyleInterpolators,
+  createStackNavigator,
+} from '@react-navigation/stack';
+import React from 'react';
+import { Platform } from 'react-native';
+import Colors from '../../constants/Colors';
+import MapScreen from '../../screens/map/MapScreen';
+import ProductDetailsScreen from '../../screens/map/ProductDetailsScreen';
+import ProductsScreen from '../../screens/map/ProductsScreen';
+import StoreListScreen from '../../screens/map/StoreListScreen';
+import RewardsScreen from '../../screens/rewards/RewardsScreen';
+
+const config = Platform.select({
+  web: { headerMode: 'screen' },
+  default: {
+    headerMode: 'none',
+  },
+});
+
+const StoresStack = createStackNavigator();
+
+export default function StoresStackNavigator() {
+  return (
+    <StoresStack.Navigator
+      screenOptions={{
```

Comment:
Gestures don't work in Android (I think they're disabled by default) so we need to explicitly enable it here (mostly for the downward swiping interaction for the rewards page).

Reply:
Hopefully this is fixed by my response to your previous comment about gesture direction!